quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/LoggingDistTest.java

```diff
@@ -71,10 +79,52 @@ void testSetLastRootLevelIfMultipleSet(LaunchResult result) {
     }
 
     @Test
-    @Launch({ "start-dev", "--log-format=\"%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c{1.}] %s%e%n\"" })
+    @Launch({ "start-dev", "--log-console-format=\"%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c{1.}] %s%e%n\"" })
```

Comment:
I'm sure you know it @DGuhr  😄  , but this is a breaking change we need to track.